content/docs/start/data-and-model-access.md

```diff
@@ -1,13 +1,16 @@
 ---
-title: 'Get Started: Data Access'
+title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data Access
+# Get Started: Data and Model Access
 
-Okay, now that we've learned how to _track_ data and models with DVC and how to
-version them with Git, next question is how can we _use_ these artifacts outside
-of the project? How do I download a model to deploy it? How do I download a
-specific version of a model? How do I reuse datasets across different projects?
+Okay, now that we've learned how to _track_ data files in DVC and how to version
+them with Git. _Models_ in a machine learning project are also files written and
+read by programs and DVC can track and version them similar to data files.
```

Comment:
Maybe "Machine learning models typically are also large files in the project. DVC can ..."

Reply:
Should this also mention "Data Science"? I'm also thinking about SEO.

Reply:
Revised in 7731587eb58c58805f5ec7c5f398dc385b1dce9c.

Reply:
> Should this also mention "Data Science"? I'm also thinking about SEO.

Was it in the desired keywords? Someone actually did research to come up with those. I see they're gone now (prob. should've left that comment until right before merging this)

Reply:
I see this still reads "Models in a machine learning project". Let's use "Machine learning models" please (SEO).

Reply:
Keywords are in BC documents, this is a GS document. There were no keywords here 😄

Reply:
> Keywords are in BC documents

True haha, nvmd.